src/test/java/sender/SenderDataImplTest.java

```diff
@@ -0,0 +1,33 @@
+package sender;
+
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import sender.SenderDataImpl;
+
+public class SenderDataImplTest {
```

Comment:
Focus on testing your functionality - the most crucial is the output format.
In the absence of a test for the output format, any future mistakes will be caught only while integrating with the receiver.